_posts/2022-10-20-login-issues.md

```diff
@@ -60,4 +60,4 @@ This is a simple `sed` command that adds the two characters to the webpacked JS
 sed -i 's/is():where()/is(*):where(*)/' /usr/share/cockpit/static/login.js
 ```
 
-It works on every distribution that has Cockpit's `login.js` in `/usr/share/cockpit/static/`. If your distribution has the file elsewhere, then change the path accordingly.
\ No newline at end of file
+It works on every distribution that has Cockpit's `login.js` in `/usr/share/cockpit/static/`. If your distribution has the file elsewhere, then change the path accordingly.
```

Comment:
I made the change on GitHub directly, so I guess it changed the last line and shows up with the last line with content. :shrug: